app/views/directives/create-secret.html

```diff
@@ -0,0 +1,307 @@
+          spellcheck="false"
+          required>
+      </div>
+      <div class="help-block">
+        Password or token for SCM servers authentication.
+      </div>
+    </div>
+  </div>
+
+  <div ng-if="newSecret.authType === 'SSH Key'">
+    <div class="form-group" id="private-key">
+      <label for="privateKey" class="required">SSH Private Key</label>
+      <osc-file-input 
+        id="privateKey"
+        model="newSecret.data.privateKey"
+        drop-zone-id="private-key"
+        dragging="false"
+        help-text="Upload your private SSH key file."
+        show-values="false"></osc-file-input>
+      <div ui-ace="{
+        theme: 'eclipse',
+        onLoad: aceLoaded,
+        rendererOptions: {
+          fadeFoldWidgets: true,
+          showPrintMargin: false 
+        }
+      }" ng-model="newSecret.data.privateKey" class="create-secret-editor ace-bordered" id="private-key-editor" required></div>
+      <div class="help-block">
+        Content of private SSH key file for SCM servers authentication.
+      </div>
+    </div>
+  </div>
+  
+  <div ng-if="newSecret.type === 'source'">
+    <div class="form-group">
+      <div class="checkbox">
+        <label>
+          <input type="checkbox" ng-model="addGitconfig">
+          Provide custom .gitconfig 
+          <span class="help action-inline">
+            <a href="">
+              <i class="pficon pficon-help" aria-hidden="true"
+              data-toggle="tooltip" data-original-title="Attach a content of your existing .gitconfig, to autheticate against Git server.">
+            </i>
+            </a>
+          </span>
+        </label>
+      </div>
+    </div>
+
+    <div class="form-group" id="gitconfig" ng-show="addGitconfig">
+      <label for="gitconfig">Git Configuration File</label>
+      <osc-file-input 
+        id="gitconfig"
+        model="newSecret.data.gitconfig"
+        drop-zone-id="gitconfig"
+        dragging="false"
+        help-text="Upload your .gitconfig or  file."
+        show-values="false"></osc-file-input>
+      <div ui-ace="{
+        mode: 'ini',
+        theme: 'eclipse',
+        onLoad: aceLoaded,
+        rendererOptions: {
+          fadeFoldWidgets: true,
+          showPrintMargin: false 
+        }
+      }" ng-model="newSecret.data.gitconfig" class="create-secret-editor ace-bordered" id="gitconfig-editor"></div>
+    </div>
+  </div>
+
+  <div ng-if="newSecret.authType === 'Docker Registry Credentials'">
+    <div class="form-group">
+      <label for="dockerServer" class="required">Docker Registry Server Address</label>
+      <div>
+        <input class="form-control"
+          id="dockerServer"
+          name="dockerServer"
+          ng-model="newSecret.data.dockerServer"
+          type="text"
+          autocorrect="off"
+          autocapitalize="off"
+          spellcheck="false"
+          required>
+      </div>
+      <div class="help-block">
+        Address of the registry that you will authenticate against.
```

Comment:
If we're repeating the field label and not adding any additional information, I'd remove the help text. It's only helpful if we say something that isn't obvious from the field name.